inductiva/coastal/bathymetry.py

```diff
@@ -328,8 +328,9 @@ def plot(
         cmap: Optional[str] = None,
         clim: Optional[Tuple[float]] = None,
         path: Optional[str] = None,
-        x_resolution: float = None,
-        y_resolution: float = None,
+        x_resolution: float = 10,
+        y_resolution: float = 10,
+        resize: bool = False,
```

Comment:
This seems very cumbersome for the user, because If I create a non-uniform grid I would be expecting to plot it as a non-uniform grid.

However, I know that this may take a few more iterations to reach there... WDYT?

Reply:
Agree with you, albeit the situation is a bit more tricky. There are non-uniform grids that are non-uniform due to the data being conveyed encoded in the latitude-longitude. This is non-uniform, but the user does not want to plot with the earth's curvature afaik. So (for now) I think we can leave it like this.

Reply:
Cool! That is different... it's a projection that needs to happen to a plane.